conf/messages.en

```diff
@@ -427,23 +427,16 @@ partyTypeMismatch.body.p2 = Check that you have chosen the right entity type and
 
 detailsDoNotMatch.title = The details you have entered do not match our records
 detailsDoNotMatch.heading = The details you have entered do not match our records
-detailsDoNotMatch.telephone.uk.label = Telephone:
-detailsDoNotMatch.textphone.label = Textphone:
-detailsDoNotMatch.telephone.intl.label = Outside UK:
-detailsDoNotMatch.open.label = Opening times:
-detailsDoNotMatch.open = Monday to Friday, 8am to 6pm
-detailsDoNotMatch.closed = Closed on weekends and bank holidays.
-
-detailsDoNotMatch.ct.body.p1 = Check your company''s Corporation Tax records and try again.
-detailsDoNotMatch.ct.body.p2 = If you are still having problems, contact the Corporation Tax helpline.
+
+detailsDoNotMatch.ct.body.p1 = Check your Self Assessment records and try again.
+detailsDoNotMatch.ct.body.p2 = Check your office address postcode matches the postcode registered with Companies House.
+detailsDoNotMatch.ct.body.p3 = If you are still having problems, you can find your Unique Taxpayer Reference, National Insurance details and tax code information in your HMRC online account.
 detailsDoNotMatch.ct.telephone.uk.number = 0300 200 3410
```

Comment:
The CT telephone numbers appear to have been left here in the messages file but no longer referenced in the view?

Reply:
Done, thank you :)